media/Config/importConfig.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @brief oneImportTools will import ONE-import tool of cfg file to webview. so user can update .cfg
+ * file comfortably.
```

Comment:
Is this like, read the cfg file and show them in the WebView so that users can selected with ComboBox or Dropdown Listbox ?

Reply:
`ONE-import tool` --> `one-import-xxx` items ?

Reply:
yes it works like that.

Reply:
`import ONE-import tool of cfg file` looks like `import onecc/one-build cfg file`
the `.cfg` files are for primary for `onecc` or `one-build` tool
and secondary for `one-xxx` tools including `one-import-xxx` tools.

Reply:
so... it means importing one-xxx tools is not necessary?

Reply:
> so... it means importing one-xxx tools is not necessary?

I mean, I think instead of `ONE-import tool of cfg `, `import onecc/one-build cfg file` would be better to explain.

Reply:
Current comment looks OK :)